studio/components/layouts/DocsLayout/DocsLayout.tsx

```diff
@@ -4,7 +4,7 @@ import { ReactElement, useEffect } from 'react'
 
```

Comment:
actually we probably can remove this file already - since the API page has been removed in favor of the side panel